CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog
 
+## HEAD
+
+- add caution icon
```

Comment:
This icon needs an entry here: https://github.com/mapbox/maki/blob/master/layouts/all.json!